pkg/components/util/types.go

```diff
@@ -78,3 +78,27 @@ func (n *NodeSelector) Render() (string, error) {
 
 	return string(b), nil
 }
+
```

Comment:
We should not add more things to `util` package. Just `pkg/components` will be fine. See #290 for more details.

Reply:
Moved to `pkg/types`.

Reply:
Fair enough :smile: